rpcserver.go

```diff
@@ -1747,15 +1747,26 @@ func (r *rpcServer) DisconnectPeer(ctx context.Context,
 	// In order to avoid erroneously disconnecting from a peer that we have
 	// an active channel with, if we have any channels active with this
 	// peer, then we'll disallow disconnecting from them.
-	if len(nodeChannels) > 0 {
-		return nil, fmt.Errorf("cannot disconnect from peer(%x), "+
-			"all active channels with the peer need to be closed "+
-			"first", pubKeyBytes)
+	if len(nodeChannels) != 0 {
+		// We are in a dev environment, print a warning log and
+		// disconnect.
+		if r.cfg.Dev.GetUnsafeDisconnect() {
```

Comment:
nit: invert `if` condition to avoid needing `else` clause?

Reply:
nice catch!